FreeRTOS/FreeRTOS-Kernel
# Normalize files with mixed line endings (introduced in commit 3a413d1)

Normalize files with mixed line endings (introduced in commit 3a413d1)

Description
-----------
Fix mixed line endings that were introduced in commit 3a413d1 

Test Steps
-----------


Related Issue
-----------
#339


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
